### DISCUSSION TITLE
Where and Why would I use RustPython

### DISCUSSION THREAD

**Discussion by krstp** [RustPython/RustPython#5424] [General]
So general question, where and why would should I prefer using RustPython. First thought that comes to my mind is possibility to circumvent GIL (like Jython does through JVM), but this is not the case, as I read somewhere RustPython will maintain GIL compatibility with Cython. What would/are the other benefits to sticking to RustPython?

I know initial Stackless release did go around GIL, but AFAIK recent releases went back to GIL; something to keep in mind if one day RustPython wanted to go this route.

In short. Convince me to try to stick to RustPython :)
